index.bs

```diff
@@ -536,6 +536,25 @@ i.e. consistency within CSS.
 
 There is also a separate section on <a href="#naming-consistency">naming consistency</a>.
 
+<h3 id="specs-include-guidance">Follow guidance from feature specifications</h3>
+
+When using a feature,
+follow guidance in the specification for that feature.
+
+Specifications for features
+that will be used by other specifications
+should include guidance for using those features.
+This ensures that the feature is used correctly
+and consistently
+throughout the platform.
+
+An incomplete list of such guidance follows:
+
+* [[encoding#specification-hooks|Encoding]]
+* [[fetch#fetch-elsewhere|Fetch]]
+* [[hr-time#sec-tools|Time]]
+* [[url#url-apis-elsewhere|URLs]]
+
```

Comment:
Maybe add something along these lines:

> Also strongly consider asking feedback from the relevant specification community to better inform your own work and ensure it's aligned with the latest best practices. Often this as easy as filing a GitHub issue.

Reply:
Or

> Consult with the relevant community
> when using their specification.
> This informs that community about new usage
> and creates opportunities for improvements
> to both the using and used specifications.

This avoids indirect acknowledgment that the documentation for usage might not be tracking best practices (when does it ever?).  I'm going for even more indirect, though I think that the goal of mutual benefit is a better angle.

The GitHub issue thing is a detail that is relevant only in some communities.

Reply:
Done.